content/20.results.md

```diff
@@ -89,9 +89,15 @@ Interaction terms did not predict the group of scientists over and above the mai
 
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/region_breakdown.png){#fig:region_breakdown}
 
-When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.18, $\beta$~East\ Asian~ = -1.71, _p_ = 6.5×10^-4^).
-We observed no statistically significant difference between the proportion of honors given to authors with Celtic/English names, European names, or names in Other categories (_p_ = 0.61, _p_ = 0.12, and _p_ = 0.72, respectively).
-Please see [analysis notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html) for more details.
+When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.17, $\beta$~East\ Asian~ = -1.77, _p_ = 4.3×10^-4^).
+We observed no statistically significant difference between the proportion of honors given to authors with Celtic/English names, European names, or names in Other categories (_p_ = 0.60, _p_ = 0.11, and _p_ = 0.70, respectively).
+Please see the [US analysis notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html) for more details.
+
+An alternative approach that considered each pair of forename and last name as the unit of measurement showed similar results.
+In addition, if we assume that honors accrue 10 years after their most prolific year with respect to authorships,
+a 10-year lag model results in a similar underrepresentation of East Asian scientists' names in the group of honorees, though the effect size is less striking.
+For example, the proportion of honor associated with East Asian name origins in 2019 is still substantially less than the proportion of senior authorships associated with East Asian names in 2009.
+Please see the [analysis notebook](https://greenelab.github.io/iscb-diversity/11.name-origin.html) for more details.
```

Comment:
One more thing - I wonder if we should do this before we dive into the US-affiliated authors. That is a deeper analysis, so we're ending the section on a weaker setup that we think is flawed instead of the stronger one.

Reply:
Good point. I'll rearrange.